lib/metasploit/framework/compiler/mingw.rb

```diff
@@ -23,14 +23,14 @@ def build_cmd(src)
           link_options = '-Wl,'
 
           src_file = File.basename(self.file_name, '.exe')
```

Comment:
The source files are still being written to the Msf path.

```suggestion
          src_file = self.file_name.split('.exe').first
```